packages/progress-circle/src/ProgressCircle.ts

```diff
@@ -101,6 +114,12 @@ export class ProgressCircle extends SizedMixin(SpectrumElement, {
         `;
     }
 
+    private onSlotChange(): void {
+        if (!this.label && this.slotHasContent) {
+            this.label = this.slotEl.assignedNodes()[0].textContent || '';
+        }
+    }
+
```

Comment:
Below in line 137, we were previously only setting `aria-label` that's technically incorrect. Can you set or remove the attribute based on the presence of `this.label` instead?